src/components/App.tsx

```diff
@@ -1279,7 +1277,7 @@ class App extends React.Component<AppProps, AppState> {
       // bounding rects don't work in tests so updating
       // the state on init would result in making the test enviro run
       // in mobile breakpoint (0 width/height), making everything fail
-      process.env.NODE_ENV !== "test"
+      !isTestEnv()
```

Comment:
It doesn't matter whether we use function or an expression since they are not pre evaluated by the bundler - they are evaluated at runtime  (I checked the webpack bundle as well to confirm) 
so in that case I think its better to use function instead to avoid code repetition.

cc @dwelle